buildtools/shrinkwrap/shrinkwrap.js

```diff
@@ -22,27 +22,28 @@ var _ = require('lodash'),
 
 module.exports = function (grunt) {
 
-  function removeResolved(json) {
+  function removeResolvedAndFrom(json) {
```

Comment:
Yeah we should definitely remove this if we have installsw, especially if that was the cause of all the problems (i.e. we should never be running this script anymore now that installsw is in)